boards/esp8266-sparkfun-thing/include/board.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_esp8266-sparcfun-thing Sparkfun ESP8266 Thing
```

Comment:
typo here.

sed s/sparcfun/sparkfun

Reply:
ping @gschorcht

Reply:
@jia200x changed it already :confused:

Reply:
oh sorry, GH was showing me the old diff

Reply:
@jia200x No problem, I fixed it only some hours ago :wink: I had not realized that it was still open until @tcschmidt was asking for. I thought your outstanding change request is related to ```new_i2c```.